main.js

```diff
@@ -46,39 +62,57 @@ function createCard(idea) {
               <label class="button-label-edit" for="">Comment</label>
             </div>
           </article>`
-}
+};
 
 function loadCards() {
   for (var i = 0; i < localStorage.length; i++) {
     var key = localStorage.key(i);
     var object = localStorage.getItem(key);
-    var parsedObject = JSON.parse(object);
-    allCards[i] = parsedObject;
+    var card = JSON.parse(object);
+    var instanciatedIdea = new Idea(card);
+    allCards[i] = instanciatedIdea;
+  };
+  orderLoadedCards();
+};
 
-    var cardHTML = createCard(parsedObject);
+function starLoad(idea) {
+  if (idea.starred) {
+    return "assets/star-active.svg";
+  } else {
+    return "assets/star.svg";
   }
+}
+
+function orderLoadedCards() {
+  allCards.sort(function(a, b) {
+    return a.id - b.id;
+  });
+  for (var i = 0; i < allCards.length; i++) {
+    createCard(allCards[i]);
+  };
 };
 
-cardSection.addEventListener('click', function() {
-    //delete handing helper function
-  if (event.target.id === 'delete') {
-    event.target.parentNode.parentNode.parentNode.remove();
-    cardRemove(event);
-    //star change helper function
-  } else if (event.target.id === 'star') {
-    console.log(event);
-    var src = event.target.src;
-    toggleStar(event);
-    if (src.includes('active')) {
-      event.target.src = "assets/star.svg";
-    } else {
-      event.target.src = "assets/star-active.svg";
-    }
-  }
-});
+function cardEvent(passedEvent) {
```

Comment:
sort of confusing function name, something like handleCardEvent might be more explicit (I tend to add a handle in front of my event listeners for this reason)